doc/HACKING

```diff
@@ -91,10 +91,10 @@ function and add it to the list in src/fuzz/meson.build.
 Whenever possible, a seed corpus and a dictionary should also be added with new
 fuzz targets. The dictionary should be named src/fuzz/fuzz-foo.dict and the seed
 corpus should be built and exported as $OUT/fuzz-foo_seed_corpus.zip in
-scripts/oss-fuzz.sh.
+tools/oss-fuzz.sh.
```

Comment:
`oss-fuzz` expects `oss-fuzz.sh` to be in `scripts`, so https://github.com/google/oss-fuzz/blob/master/projects/systemd/build.sh#L18 should be changed too.

Reply:
Yes, I'll submit a PR there after the systemd part is merged. They are very quick in merging stuff, so it shouln't be a problem.